cumulusci/cumulusci.yml

```diff
@@ -528,6 +566,8 @@ flows:
                 flow: deploy_unmanaged
             3:
                 flow: config_dev
+            4:
```

Comment:
Why snapshot_changes here when it's against a non-scratch org (packaging)?

Reply:
@jlantz I think you misread the diff; this is in `dev_org`.